registration/include/pcl/registration/registration.h

```diff
@@ -603,6 +603,17 @@ namespace pcl
     private:
       /** \brief The point representation used (internal). */
       PointRepresentationConstPtr point_representation_;
+      
+      
+      /** \brief Stores the distances between input pointcloud and matching points on registration */
+      std::vector<float> tf_points_distance_;
+    
+    public:
+      /** \brief returns the distances between input pointcloud and matching points on registration */
+      std::vector<float> getPointsDistance() {
```

Comment:
`tf_points_distance_` is a side-product of `getFitnessScore` and is invalid if that function was not called. This needs to be documented. But I should say I'm not convinced by this design anyway. What if we add an overload for `getFitnessScore` that additionally accepts a vector and populates it with computed distances?

Reply:
`const std::vector<float>& getPointsDistance() const`